backend/cli.py

```diff
@@ -228,6 +228,8 @@ def update_media(chunk_size: int, popularity: float = 1):
                 )
                 if res.status_code != 200:
                     echo_warning(res.text)
+                else:
+                    print(res.text)
```

Comment:
For debugging - Should be removed or done in a smarter way (the sum of added media or smth)